ui-v2/app/styles/components/product.scss

```diff
@@ -0,0 +1,51 @@
+@import './product/index';
+@import './filter-bar';
+@import './form-elements/index';
+html body > svg {
+  display: none;
+}
+html.ember-loading body > svg {
+  @extend %brand-loader;
+}
+html.template-loading main > div {
+  @extend %loader;
+}
```

Comment:
These are broad selectors, and I wouldn't expect to find them in `components/product.scss` if I had to go searching.

Reply:
Sourcemaps would probably help you there 😄 see #4418 

I'm still only 90% on the `product/` folder is basically the 'general window' might choose a better name for that at some point.